### PR TITLE
Cash 914

### PR DESCRIPTION
**Pull request by BoulderBrains**

### CODE REVIEW

File: src/pages/Search/MadLibsSearch.vue [hunk 1]
```diff
@@ -264,10 +310,47 @@ export default {
 
 .search-statement-wrapper {
 	margin-bottom: rem-calc(32);
+	padding: 0.375rem;
+}
+
+.madlibs-dropdown {
+	background-image: url('../../assets/images/medium-chevron2x.png');
```
Comment (mcstover) [inline src/pages/Search/MadLibsSearch.vue]: please update to you relative syntax:
`ex. background-image: url('~@/assets/images/whatever-file-name.ext');`

File: src/pages/Search/MadLibsSearch.vue [hunk 2]
```diff
@@ -264,10 +310,47 @@ export default {
...
+	background-repeat: no-repeat;
+	background-position: 100% 65%;
```
Comment (mcstover) [inline src/pages/Search/MadLibsSearch.vue]: this seems fragile or at least prone to change across various screen widths...

File: src/components/Checkout/LoanPrice.vue [hunk 3]
```diff
@@ -255,7 +255,7 @@ export default {
 	width: rem-calc(132);
 	border-radius: $button-radius;
 	height: rem-calc(50);
-	background-image: url('../../assets/images/customDropdown.png');
+	background-image: url('~@/assets/images/customDropdown.png');
 	background-size: 2rem 2rem;
```
Comment (BoulderBrains) [inline src/components/Checkout/LoanPrice.vue]: verified this is working

File: src/components/Checkout/KivaCardRedemption.vue [hunk 4]
```diff
@@ -56,21 +56,21 @@
...
+									src="~@/assets/images/checkout/emailcard_codelocation.jpg"
 									width="450"
 								>
 								<p>Postal delivery:</p>
 								<img alt="postal-kiva-card"
 									class="postal-card"
 									height="158"
-									src="../../assets/images/checkout/physicalcard_codelocation.jpg"
+									src="~@/assets/images/checkout/physicalcard_codelocation.jpg"
 									width="430"
```
Comment (BoulderBrains) [inline src/components/Checkout/KivaCardRedemption.vue]: I've verified all these changes too, all images still in place in working.

Comment (mcstover) [reply src/components/Checkout/KivaCardRedemption.vue]: 👍

File: src/components/Checkout/LoanPrice.vue (hunk 3 above)
Comment (mcstover) [reply src/components/Checkout/LoanPrice.vue]: 👍

**Pull request merged**